enterprise/kernel/src/test/java/org/neo4j/graphdb/store/id/RelationshipIdReuseStressIT.java

```diff
@@ -83,18 +86,25 @@ public void relationshipIdReused() throws Exception
 
         AtomicBoolean stopFlag = new AtomicBoolean( false );
         RelationshipsCreator relationshipsCreator = new RelationshipsCreator( stopFlag, bandLabel, cityLabel );
+        RelationshipRemover relationshipRemover = new RelationshipRemover( bandLabel, cityLabel, stopFlag );
+        IdController idController = embeddedDatabase.getDependencyResolver().resolveDependency( IdController.class );
 
-        List<Future> futures = new ArrayList<>();
-        futures.add( startRelationshipRemoval( bandLabel, cityLabel, stopFlag ) );
-        futures.add( startRelationshipsCreator( relationshipsCreator ) );
+        List<Future<?>> futures = new ArrayList<>();
+        futures.add( executorService.submit( relationshipRemover ) );
```

Comment:
It looks a bit strange you only inlined half of the methods. Otherwise looks good!

Reply:
Inlined the ones that were needed only (where the instances was created before calling the method).